resources/langs/nheko_en.ts

```diff
@@ -533,53 +515,71 @@
         <source>Save file</source>
         <translation>Save file</translation>
     </message>
+    <message>
+        <location line="-582"/>
+        <source>-- Encrypted Event (No keys found for decryption) --</source>
+        <comment>Placeholder, when the message was not decrypted yet or can&apos;t be decrypted.</comment>
+        <translation type="unfinished">-- Encrypted Event (No keys found for decryption) --</translation>
+    </message>
+    <message>
+        <location line="+46"/>
+        <source>-- Decryption Error (%1) --</source>
+        <comment>Placeholder, when the message can&apos;t be decrypted. In this case, the Olm decrytion returned an error, which is passed ad %1.</comment>
+        <translation type="unfinished">-- Decryption Error (%1) --</translation>
+    </message>
+    <message>
+        <location line="+29"/>
+        <source>-- Encrypted Event (Unknown event type) --</source>
+        <comment>Placeholder, when the message was decrypted, but we couldn&apos;t parse it, because Nheko/mtxclient don&apos;t support that event type yet.</comment>
+        <translation type="unfinished">-- Encrypted Event (Unknown event type) --</translation>
+    </message>
     <message numerus="yes">
-        <location line="+129"/>
-        <source>%1 and %2 are typing</source>
+        <location line="+636"/>
+        <source>%1 and %2 are typing.</source>
         <comment>Multiple users are typing. First argument is a comma separated list of potentially multiple users. Second argument is the last user of that list. (If only one user is typing, %1 is empty. You should still use it in your string though to silence Qt warnings.)</comment>
-        <translation>
-            <numerusform>%1%2 is typing</numerusform>
-            <numerusform>%1 and %2 are typing</numerusform>
+        <translation type="unfinished">
+            <numerusform></numerusform>
+            <numerusform></numerusform>
```

Comment:
From the diff it looks like the numerus forms vanished here, could you add them back? The update call probably messed that up.